tests/spark/test_spark_model_export.py

```diff
@@ -330,7 +330,10 @@ def test_container_scoring_responds_to_bad_inputs_using_error_message_with_mleap
                       sample_input=spark_model_iris.training_df,
                       mlflow_model=mleap_model)
     assert mleap.FLAVOR_NAME in mleap_model.flavors
-    mleap_response = score_model_in_sagemaker_docker_container(model_path=model_path,
-                                                               data="invalid")
-    assert "Error" in mleap_response.keys()
-    print(mleap_response["Error"])
```

Comment:
I am actually planning to remove this test in a subsequent PR.  I don't think it is necessary.

I would prefer to include the description in a doc string instead of a very long name.

Reply:
Agreed. Let's remove the test.